app/view_models/sign_up_completions_show.rb

```diff
@@ -54,7 +54,7 @@ def heading
 
   def title
     if requested_ial == 'ial2'
-      I18n.t('titles.sign_up.verified')
+      I18n.t('titles.sign_up.verified', app: APP_NAME)
```

Comment:
I think this is supposed to be `service_provider.friendly_name` instead of `APP_NAME`